src/solver/SolverLayout.tsx

```diff
@@ -149,6 +151,12 @@ export const SolverLayout = (props: SolverLayoutProps) => {
   const nodesInitialized = useNodesInitialized();
   const [initialLayoutFinished, setInitialLayoutFinished] = useState(false);
   const [initialFitViewFinished, setInitialFitViewFinished] = useState(false);
+  const [isFullscreen, setIsFullscreen] = useState(false);
```

Comment:
We can use `const [isFullscreen, { toggle: toggleIsFullscreen }] = useDisclosure(false);` from Mantine, elseway it would be better to do `setIsFullscreen(prevIsFullscreen => !prevIsFullscreen);` but I find it verbose

Reply:
What do you mean with verbose ? :)

Reply:
Oh just "long", sorry; it's just my bikeshedding, feel free to use the one you prefer; the only important bit is to not use `setState` without the callback (`setState(s => ...)`